packages/frontend/src/common/components/Footer.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Box, Link, Paper, useTheme } from "@mui/material";
+
+export default function Footer() {
+  const theme = useTheme();
+  return (
+    <footer>
+      <Paper
+        elevation={0}
+        square
+        sx={{
+          position: "fixed",
+          bottom: 0,
+          width: "100%",
+          backgroundColor: theme.palette.background.paper,
+          height: theme.spacing(4),
+        }}
```

Comment:
property styles available?

Reply:
For the MUI Paper, there are no property styles available